modules/dists/dims/BlockCycDim.chpl

```diff
@@ -22,6 +22,9 @@
 // Block-cyclic dimension specifier - for use with DimensionalDist2D.
 //
 
+@unstable("BlockCycDim is intended for use with DimensionalDist2D, which is unstable")
+prototype module BlockCycDim {
+
```

Comment:
You should make sure the docs don't get messed up by this change

Reply:
All looks well ✅